### DISCUSSION TITLE
How to find a noisy neighbor?

### DISCUSSION THREAD

**Discussion by skreddy6673** [concourse/concourse#6012] [Help & Support]
How to determine which team/Pipeline is utilizing most resources?

**Comment by taylorsilva** [comment]
I don't have an answer to your question right now, just putting this on your radar.
@jamieklassen is working on a tool, [`ft`](https://github.com/concourse/ft), that should (I think?) answer this question eventually. I see cpu and memory as stats that `ft` will eventually output. I'd suggest keeping an eye on it :)

**Reply by chenbh** [reply]
It looks like memory usage is 5th on the todo list https://project.concourse-ci.org/projects/MDc6UHJvamVjdDQ4NzAyMDU=

**Comment by skreddy6673** [comment]
is there anyway to see number of containers being created for a pipeline/Team?

**Reply by DanielJonesEB** [reply]
Hey @skreddy6673, at EngineerBetter we had a proof-of-concept integration of WeaveScope for Concourse, which would present a graphical view of all processes on all workers, how much CPU and RAM they were using, and which team, pipeline and job they belonged to.

We haven't maintained it for several years, but if there was an enterprise user willing to pay for it, it's something that we could work on.

![WeaveScope for Concourse](https://pbs.twimg.com/media/DxrsGj6UUAI6mZF?format=jpg&name=large)
